### ISSUE TITLE
Incineration chamber UI sometimes doesn't correctly reflect gases present inside

### ISSUE THREAD

**Issue by SyncIt21** [tgstation/tgstation#70074]
Reporting client version: 514.1588

 **Round ID:N/A**

**Frequency: Happens mostly when gases react inside the chamber**

**Reproduction**
1) Send some o2 into the incineration chamber

2) Send some plasma into the chamber

3) Ignite it to produce co2 and h20

4) Use scrubber to remove  gases from the chamber

5) H20 seems to still be present in the chamber even though the monitor tells its not
![Screenshot (14)](https://user-images.githubusercontent.com/110812394/191758959-0663cdb3-03b7-4c08-8843-fc94c38e8503.png)

Mostly seems to happen after gases react in the chamber and the UI doesn't update  even after removing those gases completely. Even with the chamber blast doors open it doesn't make a difference

Another example is when proto nitrate is made inside the chamber by igniting Pluxonium + O2 + H2 .

**Comment by Jackal-boop** [comment]
I've noticed this bug Aswel

**Comment by vincentiusvin** [comment]
Did the gases register before the burn?

**Comment by SyncIt21** [comment]
Yes they did.  It's only after the burn and scrubbing does this bug occur.  If new gases are pumped in the chamber afterwards it fixes itself. 

Another observation  I made is that sometimes the gas sensor shows  water vapor 0%(0.00 moles) in the reading regardless of scrubbing and opening blast doors.

Exact circumstances when this happens is random but again occurs mostly with gases that react in the chamber . Maybe their related?

**Comment by vincentiusvin** [comment]
Ok I digged around a bit and found out that it has nothing to do with the gas monitors/chamber, its just overlays failing to update.
![image](https://user-images.githubusercontent.com/54709710/192448322-06bb8452-9b2b-4d4e-82c4-20e029f7e8c6.png)
There is no h2o left on the tile, but the overlay hangs around. VV is of the bottom left tile.

Call update_visuals() once and its fixed.
![image](https://user-images.githubusercontent.com/54709710/192448442-9d46a7c2-9ee6-4576-803e-1e1fa77f640d.png)

**Comment by Mooshimi** [comment]
@SyncIt21 could you rename this to "incinerator chamber does not update visual overlay inside the chamber after burns" or something to reflect the issue please

**Comment by SyncIt21** [comment]
Done
